README.md

```diff
@@ -26,11 +26,16 @@ by some organizations to back user interfaces for KubeRay resource management.
 
 * [AWS Application Load Balancer](docs/guidance/ingress.md)
 * [Nginx](docs/guidance/ingress.md)
-* [Prometheus and Grafana](docs/guidance/prometheus-grafana.md) 
+* [Prometheus and Grafana](docs/guidance/prometheus-grafana.md)
 * [Volcano](docs/guidance/volcano-integration.md)
 * [MCAD](docs/guidance/kuberay-with-MCAD.md)
 * [Kubeflow](docs/guidance/kubeflow-integration.md)
 
+## Blogs
+
+* [AI/ML Models Batch Training at Scale with Open Data Hub](https://cloud.redhat.com/blog/ai/ml-models-batch-training-at-scale-with-open-data-hub)
```

Comment:
Maybe we can add the name of the company and the publication date?

Reply:
@kevin85421 Sounds good. Added.